tools/windowsplitter/windowSplitter.py

```diff
@@ -10,85 +10,85 @@
 import sys
```

Comment:
I'm already working on this in https://github.com/galaxyproject/tools-devteam/pull/559

Reply:
Wonderful. Then I take this tool out of this PR? Or do you like to add your changes here?

Reply:
I'd go for the first option, the more the merrier :D

Reply:
OK. I removed them here.